### ISSUE TITLE
Cannot pull tiller image using docker 1.13.0 on CentOS 7.3

### ISSUE THREAD

**Issue by mhulscher** [kubernetes/helm#1838]
```
[root@node-005056a63f3a ~]# docker version
Client:
 Version:      1.13.0
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Tue Jan 17 09:55:28 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.13.0
 API version:  1.25 (minimum version 1.12)
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Tue Jan 17 09:55:28 2017
 OS/Arch:      linux/amd64
 Experimental: false
```

```
[root@node-005056a63f3a ~]# docker pull gcr.io/kubernetes-helm/tiller:v2.1.3
Error response from daemon: target is unknown
```

```
[root@node-005056a63f3a ~]# uname -a
Linux node-005056a63f3a 4.4.43-1.el7.elrepo.x86_64 #1 SMP Mon Jan 16 12:56:35 EST 2017 x86_64 x86_64 x86_64 GNU/Linux
[root@node-005056a63f3a ~]# cat /etc/centos-release
CentOS Linux release 7.3.1611 (Core) 
```

**Comment by titilambert** [comment]
I got the same issue on debian testing

**Comment by sdake** [comment]
One short-term workaround provided by portdirect for kolla-kubernetes (as this issue is blocking OpenStack on Kubernetes development) is:
[17:17:59]  <portdirect>	1:) kubectl delete --namespace=kube-system deploy tiller-deploy
[17:18:46]  <portdirect>	2:) helm init --tiller-image docker.io/port/tiller:v2.1.3

**Comment by sdake** [comment]
Current speculation in the OpenStack Kolla community is that docker-engine 1.13 is not compatible with dashes in the repo organization name (kubernetes-helm in this case).  This is reinforced by attempting to create a kubernetes-helm repo in called kubernetes-helm on dockerhub, which returns a red box around the name.

**Comment by thomastaylor312** [comment]
This is related to #1824. Problem is that Circle CI only supports up to Docker 1.10 and for the media type to change it needs 1.11

**Comment by thomastaylor312** [comment]
Also, this may be fixed with 1.13.1, but isn't completely clear. [Source](https://github.com/docker/for-mac/issues/1179)

**Comment by technosophos** [comment]
After some investigation, I'm closing this as a duplicate of #1824.

The current work-around is to use Docker >= 1.13.1-RC1, which fixes the bug in Docker 1.13.0
